docs/user-guide/local.md

```diff
@@ -48,7 +48,6 @@ If you get the following error while running the update command,
 Error occurred while detecting version: GET https://api.github.com/repos/screwdriver-cd/sd-local/releases: 403 API rate limit exceeded.
 ```
 Please set the [GitHub personal access token.](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token)
-### bash
```

Comment:
Is it OK to delete this line?

Reply:
Maybe we can remove this line. I don't understand why it's there.

Reply:
🙆‍♀️